mmv1/products/storage/Folder.yaml

```diff
@@ -0,0 +1,82 @@
+# Copyright 2024 Google Inc.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+---
+name: 'Folder'
+kind: 'storage#folder'
+base_url: 'b/{{bucket}}/folders'
+self_link: 'b/{{bucket}}/folders/{{%name}}'
+id_format: '{{bucket}}/{{name}}'
+delete_url: 'b/{{bucket}}/folders/{{%name}}'
+has_self_link: true
+timeouts:
+  insert_minutes: 20
+  update_minutes: 20
+  delete_minutes: 20
+exclude_sweeper: true
+import_format:
+  - '{{bucket}}/folders/{{%name}}'
+  - '{{bucket}}/{{%name}}'
+examples:
+  - name: 'storage_folder_basic'
+    primary_resource_id: 'folder'
+    vars:
+      bucket_name: 'my-bucket'
+    ignore_read_extra:
+      - 'force_destroy'
+      - 'recursive'
+description: |
+  A Google Cloud Storage Folder.
+
+  The Folder resource represents a folder in a Cloud Storage bucket with hierarchical namespace enabled
+references:
+  guides:
+    'Official Documentation': 'https://cloud.google.com/storage/docs/folders-overview'
+  api: 'https://cloud.google.com/storage/docs/json_api/v1/folders'
+# iam_policy: handwritten in mmv1/third_party/terraform/services/storage/iam_storage_managed_folder.go
+custom_code:
+parameters:
+  - name: 'bucket'
+    resource: 'Bucket'
+    imports: 'name'
+    description: 'The name of the bucket that contains the folder.'
+    required: true
+    immutable: true
+  - name: 'name'
+    description: |
+      The name of the folder expressed as a path. Must include
+      trailing '/'. For example, `example_dir/example_dir2/`.
+    required: true
+    # The API returns values with trailing slashes, even if not
+    # provided. Enforcing trailing slashes prevents diffs and ensures
+    # consistent output.
+    validation:
+      regex: '/$'
+  - name: 'recursive'
```

Comment:
We are not planning to add this field, Right?

Reply:
removed